### PR TITLE
[Gardening] Remove extra Foundation imports

### PR DESCRIPTION
**Pull request by palimondo**
Remove the import of Foundation where it is not necessary for testing the ObjC interoperability.

Edit: Also import **MSVCRT** in Windows where the `#if os(Linux) import Glibc` pattern occurs.

### CODE REVIEW

**Comment by palimondo** [comment]
@swift-ci please benchmark

**Comment by palimondo** [comment]
@swift-ci please smoke test

**Comment by palimondo** [comment]
Those regressions and improvements in `-O` and `-Osize` are unusually high for no changes… Similar to [the other case](https://github.com/apple/swift/pull/25309#issuecomment-506691875)? I wonder if these are flukes or replicable. Re-running benchmark to see.

File: benchmark/single-source/Radix2CooleyTukey.swift [hunk 1]
```diff
@@ -2,7 +2,11 @@
 //
 // Originally written by @owensd. Used with his permission.
 
-import Foundation
+#if os(Linux)
+import Glibc
+#else
+import Darwin
+#endif
```
Comment (compnerd) [inline benchmark/single-source/Radix2CooleyTukey.swift]: Could you please add:

```
#elseif os(Windows)
import MSVCRT
#else
```

File: benchmark/utils/ArgParse.swift [hunk 2]
```diff
@@ -10,7 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-import Foundation
+#if os(Linux)
+import Glibc
+#else
+import Darwin
+#endif
```
Comment (compnerd) [inline benchmark/utils/ArgParse.swift]: Similar

**Comment by jrose-apple** [comment]
There are a few things that behave differently based on whether the Foundation overlays are linked into the process. :-( The set is kept pretty small but it could still be an explanation here. cc @Catfish-Man

**Comment by palimondo** [comment]
@swift-ci please benchmark

**Review by eeckstein — APPROVED** [review]
lgtm!
(Please address compnerd's request)

**Comment by palimondo** [comment]
@swift-ci please benchmark

**Comment by palimondo** [comment]
@swift-ci please benchmark

**Comment by palimondo** [comment]
@swift-ci smoke test

**Comment by palimondo** [comment]
@swift-ci smoke test

**Pull request merged**